StartAKSClusterAssessment.ps1

```diff
@@ -65,17 +92,46 @@ foreach ($currentSubscription in $subscriptions) {
     Write-Host "***** Assessing the subscription $($currentSubscription.displayName) ($($currentSubscription.id)..." -ForegroundColor Cyan
     az account set -s $currentSubscription.SubscriptionId --only-show-errors 
 
-    $jsonAksClusters = az aks list -o json --only-show-errors 
```

Comment:
I honestly don't know what this piece of code is doing here, I believe the entire section of this red zone 68-75 should be removed